Sources/MapboxMaps/Location/LocationManager.swift

```diff
@@ -54,6 +54,10 @@ public class LocationManager: NSObject {
             if let puckType = options.puckType, puckType != oldValue.puckType {
                 locationPuckManager?.changePuckType(to: puckType)
             }
+
+            if options.puckBearingSource != oldValue.puckBearingSource {
+                locationPuckManager?.changePuckBearingSource(to: options.puckBearingSource)
```

Comment:
this should be updated.

Reply:
To what? Unless you are saying to omit the `changePuckBearingSource` function on `LocationPuckManager` as well. if that is the case, I would disagree. I think one level of encapsulation here is nice to read

Reply:
hmm let's address as part of the coming refactor.